lib/model/query/submissions.js

```diff
@@ -121,6 +121,9 @@ const clearDraftSubmissions = (formId) => ({ run }) =>
 const clearDraftSubmissionsForProject = (projectId) => ({ run }) =>
   run(sql`DELETE FROM submissions USING forms WHERE submissions."formId" = forms.id AND forms."projectId" = ${projectId} AND submissions.draft=true`);
 
+const softDeleteDraftSubmissions = (formId) => ({ run }) =>
```

Comment:
Could we name this `delete` rather than `softDelete`? I can't think of another example offhand where we use the term `softDelete` in the code itself.